### ISSUE TITLE
Misc stats for setting up / tearing down containers

### ISSUE THREAD

**Issue by atc0005** [atc0005/ansible-role-lxd-testenv#12]
## ZFS-based storage

3.1 GB for images/containers (/var/lib/lxd/)

### Re-run of `site.yml`, no needed changes

```
real	3m10.174s
user	0m26.084s
sys	0m8.208s
```

### Removal of containers (`lxd-remove.yml`)

```
real	0m40.381s
user	0m5.460s
sys	0m1.528s
```

### Fresh `site.yml` run (complete setup)

```
real	9m1.974s
user	0m39.720s
sys	0m11.784s
```

### Fresh `lxd-setup.yml` run (partial setup)

```
real	8m14.754s
user	0m37.216s
sys	0m11.300s
```

## dir based storage

6.3 GB for images/containers (/var/lib/lxd/)

### Re-run of `site.yml`, no needed changes

```
real	3m12.435s
user	0m26.972s
sys	0m9.652s
```

### Removal of containers (`lxd-remove.yml`)

```
real	0m40.351s
user	0m5.460s
sys	0m1.408s
```

### Fresh `site.yml` run (complete setup)

```
real	9m26.728s
user	0m40.732s
sys	0m12.344s
```

### Fresh `lxd-setup.yml` run (partial setup)

```
real	7m54.446s
user	0m35.020s
sys	0m10.744s
```

**Comment by atc0005** [comment]
Added to branch that I'm polishing for a v1.1 release.

**Comment by atc0005** [comment]
Included in #14.
